api/src/main/java/org/hawkular/inventory/api/Relationships.java

```diff
@@ -91,14 +117,17 @@ private Relationships() {
     /**
      * Provides read-write access to relationships.
      */
-    public interface ReadWrite extends ReadWriteInterface<Relationship, Relationship.Blueprint, Single, Multiple> {
```

Comment:
The `getMeTheOtherEndOfThisRelationshipPlease()` method could be simply replaced by `...entity().getTarget())`.